### ISSUE TITLE
ess-auto-width: problem when activating through Customize

### ISSUE THREAD

**Issue by braunm** [emacs-ess/ESS#959]
I decided to experiment with `ess-auto-width` option. (Aquamacs 3.5, ESS 18.10.3snapsot from Melpa). When I set its value to "Window width" through the customization buffer, the following is written to my customization file:

`'(ess-auto-width (quote (quote window)))`

Afterwards, after launching an R buffer, I will see the following error in the minibuffer at seemingly unrelated times, preventing me from doing simple tasks like opening files:

`OPT ((quote window)) not 'window, 'frame or an integer`.

I then removed the customization, and inserted the following in my init file:

`(setq ess-auto-width 'window)`

And now automatically setting the R width based on the ess window width works just fine.

In short, it looks like the variable does what it is supposed to do, but it is not being set correctly through Customize.

**Comment by jabranham** [comment]
Thank you for the bug report. Should be fixed.
